students/TimPauley/session06/test_MailRoomPart4.py

```diff
@@ -63,7 +64,7 @@ def test_generate_donor_report():
 
     assert report.startswith("Donor Name                | Total Given | Num Gifts | Average Gift")
 
-    assert "Paul Allen                  $    663.23           1   $     663.23" in report
+    assert "Jeff Bezos                  $    877.33           1   $     877.33" in report
```

Comment:
you could use a bit more testing here -- did all the donors make it into the report?

and can you test a row in the report without hard coding EXACTLY how it will be laid out?